pkg/vaultutil/vaultutil.go

```diff
@@ -86,32 +82,22 @@ func getVaultTfSecret(client *vault.Client, secretInfo VaultSecret, kvVersion st
 			rawSecret, err = client.Logical().Read(path)
 		}
 		if err != nil {
-			return TfCreds{}, err
+			return nil, err
 		}
 		if rawSecret == nil {
-			return TfCreds{}, fmt.Errorf("No secret found at specified path: %s", secretInfo.Path)
+			return nil, fmt.Errorf("No secret found at specified path: %s", secretInfo.Path)
 		}
 		if len(rawSecret.Data) == 0 {
-			return TfCreds{}, fmt.Errorf("No key-values stored within secret at path: %s", secretInfo.Path)
+			return nil, fmt.Errorf("No key-values stored within secret at path: %s", secretInfo.Path)
 		}
 		var ok bool
 		secret, ok = rawSecret.Data["data"].(map[string]interface{})
 		if !ok {
-			return TfCreds{}, fmt.Errorf("Failed to process data for secret at path: %s", secretInfo.Path)
+			return nil, fmt.Errorf("Failed to process data for secret at path: %s", secretInfo.Path)
 		}
 	default:
-		return TfCreds{}, fmt.Errorf("Invalid vault kv engine version specified: %s", kvVersion)
+		return nil, fmt.Errorf("Invalid vault kv engine version specified: %s", kvVersion)
 	}
 
-	for _, key := range []string{AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY, AWS_BUCKET, AWS_REGION} {
```

Comment:
logic specific to terraform expected credentials. This has been moved to terraform.go